src/Compilers/Test/Utilities/VisualBasic/SemanticModelTestBase.vb

```diff
@@ -151,4 +152,32 @@ Public MustInherit Class SemanticModelTestBase : Inherits BasicTestBase
             ).Where(Function(s) anyArity OrElse DirectCast(s, Symbol).GetArity() = arity.Value).ToList()
     End Function
 
+    Friend Function GetOperationTreeForTest(Of TSyntaxNode As SyntaxNode)(compilation As VisualBasicCompilation, fileName As String, Optional which As Integer = 0) As String
```

Comment:
Yes, it seems there are differences between binding test framework helpers, and I had to retain them so I can re-use it.